src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const empoyees = [...document.querySelectorAll('li')];
+
+const dataAtributToNumber = (string) => {
+  const arr = string.split('');
+
+  arr.shift();
+
+  const strNormalized = arr.filter((el) => el !== ',').join('');
+
+  return +strNormalized;
```

Comment:
The function `dataAtributToNumber` is a bit complex and can be simplified. Instead of splitting the string into an array, shifting the first element, filtering out commas, and joining it back into a string, you can use a regular expression to replace all non-digit characters. For example, `string.replace(/\D/g, '')`. This will make your code cleaner and more performant.